### ISSUE TITLE
Is it possible to colourise object properties the way Sublime text does?

### ISSUE THREAD

**Issue by feardamhan** [Maximetinu/Sublime-Text-Monokai-theme-for-Visual-Studio-Code#11]
Like this? 
![image](https://user-images.githubusercontent.com/35301475/72623182-8925d380-393c-11ea-9900-240d9e327f86.png)

**Comment by maximetinu** [comment]
Hi

I never knew what you meant. Is it PHP?

**Comment by feardamhan** [comment]
Sorry for delay in response. I was away for a few days. In the above example that a powershell object. the object being 'dn' and then property being 'distinguishedName'. as you can see Sublime Text differentiates between the object and property (or nested sub properties) using colour. this makes if far easier to read. Was asking if it were possible for the theme to do the same.

**Comment by maximetinu** [comment]
Do not apologize, it took me more than a year to answer your initial issue 😋 

I know what you mean now, you were referring to .ps1 scripts for PowerShell, right?

I've added support for it, I modified the color of the `$` keyword to red, the `_` to blue, and the properties to green. Apart from those differences, the rest was already equal to Sublime Text, so I'm glad to announce that the extension officially supports PowerShell scripts from now on 😁 A new 1.0.5 version is up and ready to download from the VS Code extensions tab.

Comparison:

![image](https://user-images.githubusercontent.com/16073309/117583718-b03f8000-b100-11eb-9120-ed86569a9a49.png)

(I took the liberty of cloning your powershell-scripts repo to do the comparison 😄)

Let me know if you find any other difference 👍 Cheers!
